projects/VRDR/DeathRecord_fieldsAndCreateMethods.cs

```diff
@@ -229,6 +235,44 @@ private void CreateBirthRecordIdentifier()
             Bundle.AddResourceEntry(BirthRecordIdentifier, "urn:uuid:" + BirthRecordIdentifier.Id);
         }
 
+       /// <summary>Create an empty BirthRecordIdentifierChild Observation.</summary>
+        private void CreateBirthRecordIdentifierChild()
+        {
+            BirthRecordIdentifierChild = new Observation();
+            BirthRecordIdentifierChild.Id = Guid.NewGuid().ToString();
+            BirthRecordIdentifierChild.Meta = new Meta();
+            string[] br_profile = { ProfileURL.BirthRecordIdentifier };
+            BirthRecordIdentifierChild.Meta.Profile = br_profile;
+            BirthRecordIdentifierChild.Status = ObservationStatus.Final;
+            BirthRecordIdentifierChild.Code = new CodeableConcept(VR.CodeSystems.LocalObservationCodes, "decedentbirthrecordidentifier", "Birth Record Identifier of Decedent", null);
```

Comment:
This matches the IG, but should this line and line 229 above be swapped? isn't this one the childbirthrecordidentifier? If that's the case then the IG needs to be fixed as well

Reply:
- 224: Birth Record Identifier (Infant) -- Decedent is infant.  Identifier links to infants birth certificate.
- 244: Birth Record Identifier Child -- decedent is mother.  Identifier links to a recently born child's birth certificate 
- Fetal Death Record Identifier -- decedent is mother.  Indentifier links to a recent fetal death